### DISCUSSION TITLE
145

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#81] [General]
# 潮流周刊第145期 - 小橡皮树

封面图来源于周末去了一趟杭州宜家，今年第四次去了，每次都能够买到一些小东西，这次走的时候看到这个橡皮树，长得很有生命力，就买下来了。

https://weekly.tw93.fun/posts/145-%E5%B0%8F%E6%A9%A1%E7%9A%AE%E6%A0%91

**Comment by wukaipeng-dev** [comment]
希望可以出一期介绍 RSS 应用的 😃

**Reply by tw93** [reply]
试试这个 https://ranchero.com/netnewswire/

**Reply by wukaipeng-dev** [reply]
good，值得一试
